### ISSUE TITLE
Cannot Create Remote Context using Docker CLI

### ISSUE THREAD

**Issue by groogiam** [Azure/acr#724]
**Describe the bug**
I am trying to create a remote context to the ACR instance using the documentation here. https://docs.docker.com/engine/context/working-with-contexts/. The machine I am trying to connect from is a Debian container with only the docker-cli installed. When doing so all requests to the context fail with 

```xml
<html>
<head><title>400 The plain HTTP request was sent to HTTPS port</title></head>
<body bgcolor="white">
<center><h1>400 Bad Request</h1></center>
<center>The plain HTTP request was sent to HTTPS port</center>
<hr><center>openresty</center>
</body>
</html>
```

**To Reproduce**
1. `docker login myacr.azurecr.io -u myacr -p "mypass"`
2. `docker context create myacr --docker host=https://myacr.azurecr.io:443`
3. `docker context use myacr` 
4. `docker image ls`

**Expected behavior**
I should be able to use the ACR as a remote context with the docker cli

**Screenshots**

**Any relevant environment information**
 - OS: Debian
 - Azure CLI/PowerShell/SDK version N/A
 - Docker version 24.0.7, build afdd53b
 - See post time of bug
 
**Additional context**

If any information is a concern to post here, you can create a [support ticket](https://azure.microsoft.com/en-us/support/create-ticket/) or send an email to acrsup@microsoft.com.

**Comment by terencet-dev** [comment]
Hi @groogiam, apologies for the delay here. Have you opened a [support ticket](https://learn.microsoft.com/azure/azure-portal/supportability/how-to-create-azure-support-request) for this yet?

**Comment by groogiam** [comment]
No I have not. Is that the recommended path?

**Comment by terencet-dev** [comment]
Yes, it is the recommended path. If it is in fact a bug, we will open one here for our users to track/follow.

**Comment by toddysm** [comment]
@johnsonshi this is tangentially related to authentication. I don't know whether this is still an issue but it may be worth reproducing.

**Comment by johnsonshi** [comment]
@groogiam and @toddysm, ACR does not support using an ACR registry as a remote Docker context. Remote docker contexts (`docker context create --host=<url>`) is designed for Docker Daemons running on remote virtual machines, not for container registries like ACR.

Closing this as this isn't a supported scenario.
